src/Model/SearchDocument.php

```diff
@@ -164,10 +164,11 @@ protected function searchXPath($xPath, $html)
             foreach ($nodes as $node) {
                 $nodeValues[] = $node->nodeValue;
             }
+            
+            $contents = implode("\n\n", $nodeValues);
```

Comment:
@ethanjohnstone it'd be nice to stripe tags as well ?

Reply:
Would you want that as part of this PR or should I open a separate one for it? 
PS. Any chance you could add the Hacktober tag to the repo?